Dockerfile

```diff
@@ -31,15 +32,14 @@ ENV \
 RUN groupadd --system symbolicator --gid $SYMBOLICATOR_GID \
     && useradd --system --gid symbolicator --uid $SYMBOLICATOR_UID symbolicator
 
-RUN mkdir /etc/symbolicator /data \
-    && chown symbolicator:symbolicator /etc/symbolicator /data
-VOLUME ["/etc/symbolicator", "/data"]
+VOLUME ["/data"]
+RUN mkdir /etc/symbolicator && \
+    chown symbolicator:symbolicator /etc/symbolicator /data
+ONBUILD COPY --chown=symbolicator:symbolicator config.yml /etc/symbolicator/config.yml
```

Comment:
Could you elaborate on why we need this?

Reply:
This just allows any downstream users of the image to bake in their own config, instead of using volumes. This is better because volumes are host-bound by definition whereas a config copied in like this is baked into the new image, making distribution easier and safer for consistency.

This follows the same patter from https://github.com/getsentry/docker-sentry/blob/master/git/onbuild/Dockerfile

Reply:
I don't mind this, but for my understanding: if `config.yml` is absent during the downstream build, will the build fail?

Reply:
Nope it won't (and I tested this).

>The ONBUILD instruction adds to the image a trigger instruction to be executed at a later time, when the image is used as the base for another build. The trigger will be executed in the context of the downstream build, as if it had been inserted immediately after the FROM instruction in the downstream Dockerfile.

Reply:
Just checked, there are some tricks to make this optional but none are great. We deploy these images directly and I'd argue anyone building a new image off of this should bake the config in (instead of providing it via a volume). That said if this becomes an issue, we can very easily separate this and create a new tag with the `-onbuild` suffix, which seems to be the consensus.

Will leave the final decision to you as I'm okay with both, with a bias towards simplicity.

Reply:
If we aim at simplicity, I'd suggest we omit both ONBUILD and VOLUME for `/etc/symbolicator`. This way, it's easy to use your own config for downstream images and there are no hidden requirements of having `config.yml` in the same directory.

I refreshed my understanding of VOLUME directive ([this SO thread](https://stackoverflow.com/questions/52570093/what-is-the-practical-purpose-of-volume-in-dockerfile) summarizes it nicely), and now I see no strong reason of using VOLUME basically anywhere, since it cannot be undeclared in downstream images and just makes things more complicated without any real benefit.

Reply:
Alright, removing the `ONBUILD` as any downstream user can easily add their own config file via `COPY`. No need to make that compulsory.